.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "web-client"]
 	path = web-client
 	url = https://github.com/cryostatio/cryostat-web
+	branch = main
```

Comment:
This could really bite us when branching for a release. If we forget to update this, then CI will be including commits from main and we might not notice. There is the `.` option:
`
A special value of .  is used to indicate that the name of the branch in the submodule should be the same name as the current branch in the current repository.
`
But this breaks if you have a feature branch checked out...